packages/sanity/src/core/studio/components/navbar/search/components/Instructions.tsx

```diff
@@ -1,16 +1,18 @@
 import {ControlsIcon} from '@sanity/icons'
 import {Flex, Inline, Text} from '@sanity/ui'
 import React from 'react'
+import {useTranslation} from '../../../../../i18n'
 
 export function Instructions() {
+  const {t} = useTranslation()
+
   return (
     <Flex align="center" direction="column" gap={4} paddingX={4} paddingY={5}>
       <Inline space={3}>
-        <Text muted>Use</Text>
+        <Text muted>{t('navbar.search.instructions')}</Text>
```

Comment:
👋 @rexxars I had to change this slightly from what it was before (with rendering an icon as a react component), I tried looking around and I don't see a very neat way of doing this apart from altering the text slightly. However, I might be wrong and there's a neater way to keep the text as it was originally, do you see any way?

Reply:
Discovered you can do something like this: https://stackoverflow.com/questions/61201001/using-i18next-to-replace-a-variable-with-a-reactnode-element

https://react.i18next.com/latest/trans-component

I'll replace it here with the right approach

Reply:
b0c1c40ae80646873637d08920388d12f1978b66